<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MysqlClient</class>
 <widget class="QMainWindow" name="MysqlClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Выбор отображаемых данных</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelDatabase">
           <property name="text">
            <string>База данных:</string>
           </property>
           <property name="buddy">
            <cstring>comboDatabase</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboDatabase"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelTable">
           <property name="text">
            <string>Таблица:</string>
           </property>
           <property name="buddy">
            <cstring>comboTable</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboTable"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>422</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btnConnect">
      <property name="text">
       <string>Подключение</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnStartEditing">
        <property name="text">
         <string>Редактировать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSubmitEditing">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>ОК</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancelEditing">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Откатить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkboxAutoConn">
      <property name="text">
       <string>Автоматическое подключение</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Настройки</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Формы редактирования</string>
    </property>
    <addaction name="actionEditProjects"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEditProjects">
   <property name="text">
    <string>Проекты</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
